lmctl/cli/commands/login.py

```diff
@@ -25,34 +24,37 @@
 @click.option('--api-key', default=None, help='API Key for the given user (when using "--zen")')
 @click.option('--client', 'client_id', default=None, help='Client ID')
 @click.option('--client-secret', default=None, help='Secret for the given Client ID')
+@click.option('--scope', default=None, help='scope for the given Client ID')
+@click.option('--auth-server-id', default=None, help='Okta backend authentication server for the given Client ID to generate okta auth API')
 @click.option('--token', default=None, help='Authenticate with a token instead of credentials')
 @click.option('--name', default='default', show_default=True, help='Name given to the environment saved in the configuration file')
 @click.option('--auth-address', default=None, help='Auth address required for username/password access (without client credentials). This is usually the Nimrod route in your environment.')
 @click.option('--save-creds', is_flag=True, default=False, show_default=True, help='Save the credentials instead of the token. This allows lmctl to re-authenticate later without requiring a login but your passwords will be stored as plain text in the configuration file')
 @click.option('--print', 'print_token', is_flag=True, default=False, help='Print the access token rather than saving it')
 @click.option('-y', 'yes_to_prompts', is_flag=True, default=False, show_default=True, help='Force command to accept all confirmation prompts e.g. to override existing environment with the same name')
 @click.option('--zen', 'is_zen', is_flag=True, default=False, help='Indicate that the Zen authentication method should be used (must provide --api-key)')
+@click.option('--okta', 'is_okta', is_flag=True, default=False, help='Indicate that the okta authentication method should be used')
 @mutually_exclusive_group(
     ('token', '--token'),
     mutex_with=[
         ('username', '--username'),
-        ('password', '-p, --pwd, --password'), 
+        ('password', '-p, --pwd, --password'),
         ('client_id', '--client-id'),
         ('client_secret', '--client-secret'),
         ('auth_address', '--auth-address'),
+        ('auth-server-id', '--auth_server_id'),
```

Comment:
`scope` and `is_okta` will also need to be mutually exclusive with `token`

Reply:
Addressed